src/nvim/event/process.h

```diff
@@ -43,9 +45,9 @@ static inline Process process_init(Loop *loop, ProcessType type, void *data)
     .stopped_time = 0,
     .cwd = NULL,
     .argv = NULL,
-    .in = NULL,
-    .out = NULL,
-    .err = NULL,
+    .in = {0},
```

Comment:
I prefer being explicit in such cases rather then rely on compiler zeroing things out.